### PR TITLE
Fix: Replacing empty screens with messages in myworkflow and favorite_workflow fragments

### PR DESCRIPTION
**Pull request by Hiteshgautam01**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the checks with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by Hiteshgautam01** [comment]
<h1 align= "center" >UI after changes</h1>
<h1 align="center">
  

  
</h1>

**Comment by Hiteshgautam01** [comment]
Hey @sagar15795 , i think the white background on these screens looks more good than grey. So, we will discuss about the backgrounds while discussing about the color theme for the application. Until, i think the grey is also good.

File: app/src/main/res/layout/fragment_dashboard.xml [hunk 1]
```diff
@@ -21,14 +21,35 @@
...
 
-    <TextView
-            android:id="@+id/tvNoWorkflow"
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_centerVertical="true"
+        android:layout_centerHorizontal="true"
+        android:visibility="gone"
+        android:id="@+id/tvNoWorkflow"
```
Comment (sagar15795) [inline app/src/main/res/layout/fragment_dashboard.xml]: change this id.

File: app/src/main/res/layout/fragment_dashboard.xml [hunk 2]
```diff
@@ -21,14 +21,35 @@
...
+        android:layout_marginBottom="@dimen/relative_layout_margin_bottom">
+
+        <ImageView
+            android:id="@+id/workflow_icon"
```
Comment (sagar15795) [inline app/src/main/res/layout/fragment_dashboard.xml]: use camel case for id name

File: app/src/main/res/layout/fragment_favourite_workflow_list.xml [hunk 3]
```diff
@@ -37,13 +37,37 @@
...
+        android:layout_centerVertical="true"
+        android:layout_centerHorizontal="true"
+        android:visibility="gone"
+        android:id="@+id/error_no_workflow"
+        android:layout_marginBottom="@dimen/relative_layout_margin_bottom"
+        >
+
+
+        <ImageView
+            android:id="@+id/star_icon"
```
Comment (sagar15795) [inline app/src/main/res/layout/fragment_favourite_workflow_list.xml]: use camel case naming convention for id

File: app/src/main/res/layout/fragment_favourite_workflow_list.xml [hunk 4]
```diff
@@ -37,13 +37,37 @@
...
             android:visibility="visible"/>
 
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_centerVertical="true"
+        android:layout_centerHorizontal="true"
+        android:visibility="gone"
+        android:id="@+id/error_no_workflow"
```
Comment (sagar15795) [inline app/src/main/res/layout/fragment_favourite_workflow_list.xml]: use camel case naming convention for id and correct name

File: app/src/main/res/layout/fragment_dashboard.xml (hunk 2 above)
Comment (Hiteshgautam01) [reply app/src/main/res/layout/fragment_dashboard.xml]: Hey @sagar15795 , id naming should not be in camel case according to https://github.com/ribot/android-guidelines/blob/master/project_and_code_guidelines.md#2321-id-naming

Comment (sagar15795) [reply app/src/main/res/layout/fragment_dashboard.xml]: use proper prefix.

File: app/src/main/res/layout/fragment_favourite_workflow_list.xml (hunk 3 above)
Comment (sagar15795) [reply app/src/main/res/layout/fragment_favourite_workflow_list.xml]: use proper prefix.

File: app/src/main/res/layout/fragment_favourite_workflow_list.xml (hunk 4 above)
Comment (sagar15795) [reply app/src/main/res/layout/fragment_favourite_workflow_list.xml]: use proper prefix.

**Pull request merged**